app/helpers/authentication_helper.rb

```diff
@@ -27,10 +27,10 @@ def signed_up_with(user = current_user)
     providers = Authentication::Providers.enabled_for_user(user)
 
     # If the user did not authenticate with any provider, they signed up with an email.
-    auth_method = providers.any? ? providers.map(&:official_name).to_sentence : "Email & Password"
-    verb = providers.size > 1 ? "any of those" : "that"
+    auth_method = providers.any? ? providers.map(&:official_name).to_sentence : I18n.t("helpers.authentication_helper.email_password") # rubocop:disable Layout/LineLength
+    verb = providers.size > 1 ? I18n.t("helpers.authentication_helper.any_of_those") : I18n.t("helpers.authentication_helper.that") # rubocop:disable Layout/LineLength
 
-    "Reminder: you used #{auth_method} to authenticate your account, so please use #{verb} to sign in if prompted."
+    I18n.t("helpers.authentication_helper.reminder", method: auth_method, dem: verb)
```

Comment:
`dem`?

Reply:
https://en.wikipedia.org/wiki/Demonstrative
Any suggestions are welcome but I don't think it is `verb` 😂

Reply:
Gotcha, makes sense. In that case we should probably change the variable name too. I know it's a silly name but at least once we used something like `the_thing` in the codebase. 🤣